usaspending_api/api_contracts/contracts/autocomplete/programSourceComponents/AgencyIdentifier.md

```diff
@@ -0,0 +1,42 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Agency Identifier (AID)
+
+This endpoint powers USAspending.gov's Treasury Account and Federal Account Agency Identifier (AID) component filter in the Advanced Search Program Source (TAS) filter.
+
+## Agency Identifier [/api/v2/autocomplete/accounts/aid/]
+
+Returns lists of possible AIDs matching the search string and narrowed down by the given component filters.
+
+## POST
++ Request
+    + Attributes (object)
+        + filters (required, ComponentFilters)
+
++ Response 200 (application/json)
+    + Attributes (object)
+        + results (array[AgencyResult], fixed-type)
+
+# Data Structures
+
+## ComponentFilters (object)
++ aid: `12` (required, string)
```

Comment:
Currently I have all of the filters as optional and nullable.  If you don't supply anything (`{"filters": {}}`) you'll get everything we have for a field.  If we don't want that behavior, I can tweak the endpoint.  This, of course, doesn't mean you have to implement it that way on the front end, it's just what the API currently supports.